daemon/api_system_volumes.go

```diff
@@ -87,3 +92,29 @@ func postSystemVolumesActionGenerateRecoveryKey(c *Command) Response {
 		"key-id":       keyID,
 	})
 }
+
+var fdeMgrCheckRecoveryKey = func(fdemgr *fdestate.FDEManager, rkey keys.RecoveryKey, containerRoles []string) (err error) {
+	return fdemgr.CheckRecoveryKey(rkey, containerRoles)
+}
```

Comment:
I believe this could be

```suggestion
var fdeMgrCheckRecoveryKey = fdemgr.CheckRecoveryKey
}
```

Also, should it be `fdemgr` instead of `fdeMgr` for consistency?

Reply:
Sorry, I messed up, meant this:

```suggestion
var fdeMgrCheckRecoveryKey = (*fdestate.FDEManager).CheckRecoveryKey
```

Reply:
TIL something new, Thank you!